ofrak_core/pytest_ofrak/elf/fixtures.py

```diff
@@ -6,7 +6,7 @@
 
 
 # MAKEFILE_CONTENTS = """
-# CC=gcc
+# CC=x86_64-linux-gnu-gcc
```

Comment:
Same comment as above about dynamically building ELF test assets.

Also this stuff is entirely commented out anyway and probably shouldn't even be in this file?

Reply:
Removed these lines